org.eclipse.sisu.plexus/src/main/java/org/codehaus/plexus/DefaultPlexusContainer.java

```diff
@@ -200,8 +203,7 @@ public DefaultPlexusContainer( final ContainerConfiguration configuration, final
         final ClassSpace space = new URLClassSpace( containerRealm );
         beanModules.add( new PlexusXmlBeanModule( space, variables, plexusXml ) );
         final BeanScanning global = BeanScanning.INDEX == scanning ? BeanScanning.GLOBAL_INDEX : scanning;
-        beanModules.add( new PlexusAnnotatedBeanModule( space, variables, global,
-                                                        configuration.getStrictClassPathScanning() ) );
```

Comment:
Q: why config.getStringClasspathScanning is not good? Why adding private member instead?

Reply:
I need the flag also in the other method...